app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ActiveRecord::Base
+  has_many :bookings
+  has_secure_password
+
+  validates_presence_of :name
+  validates_presence_of :email
+  validates_uniqueness_of :email
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.